### ISSUE TITLE
Show Wallets on home page.

### ISSUE THREAD

**Issue by Maniixer** [dogecoinfoundation/dogecoin.com#139]
It is very important to have the wallets that Dogecoin recommend be shown on the home page, right now it is not user friendly,

so i made a suggestion on what could be a better solution for this section.
![image](https://user-images.githubusercontent.com/92598085/181224008-b246dc18-105d-41db-af90-8638dc843492.png)

**Comment by Maniixer** [comment]
![image](https://user-images.githubusercontent.com/92598085/181226488-48eb1c70-86d1-43a5-ac44-34fdcb7d0ff9.png)

**Comment by alamshafil** [comment]
The website actually did show the wallets before but it didn't provide information if it was closed-source, self-custodian, etc.  There was a discussion about this on Twitter regrading the legal complications if someone used the wallet recommend and somehow lost their coins and blaming the Dogecoin team for it. I believe @qlpqlp can give you information into this!

**Comment by Maniixer** [comment]
https://foundation.dogecoin.com/website.png

Yes this design is really good tbh, so i'm not understanding why they are not following it :p 
they can just add the information on it instead of completely removing it and placing it on a different page that looks like a data base of a back-end developer xd

**Comment by MishaBoar** [comment]
> The website actually did show the wallets before but it didn't provide information if it was closed-source, self-custodian, etc. There was a discussion about this on Twitter regrading the legal complications if someone used the wallet recommend and somehow lost their coins and blaming the Dogecoin team for it. I believe @qlpqlp can give you information into this!

Yeah, I agree this is important: https://github.com/dogecoinfoundation/dogecoin.com/issues/132#issue-1317104642

**Comment by qlpqlp** [comment]
What wallets should be shown on the first page because they are many and will grow with time :)

**Comment by qlpqlp** [comment]
> > The website actually did show the wallets before but it didn't provide information if it was closed-source, self-custodian, etc. There was a discussion about this on Twitter regrading the legal complications if someone used the wallet recommend and somehow lost their coins and blaming the Dogecoin team for it. I believe @qlpqlp can give you information into this!
> 
> Yeah, I agree this is important: [#132 (comment)](https://github.com/dogecoinfoundation/dogecoin.com/issues/132#issue-1317104642)

About this, on the bitcoin org website they do not list any wallet, only on bitcoin.com and only list their own wallet along with options to buy/sell bitcoin

Do you recommend following the bircoint.org or bitcoin.com ways of do it @MishaBoar ?
